server/config.js

```diff
@@ -4,11 +4,15 @@ dotenv.config();
 const envConfig = {
     development: {
         dbHost: process.env.MONGO_URL || 'mongodb://localhost/snappit',
-        port: process.env.PORT || 3000
+        port: process.env.PORT || 3000,
+        jwtSecret: 'thisisasecret',
+        jwtExpiresIn: '7d'
     },
     production: {
         dbHost: process.env.MONGO_URL,
-        port: process.env.PORT || 3000
+        port: process.env.PORT || 3000,
+        jwtSecret: 'qwertyuiop1234567890',
```

Comment:
This should be a part of env variables, right?

Reply:
it is taking from process.env. 3000 is just added as default in case variable is not provided in environment